packages/hash/task-executor/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "@hashintel/task-executor",
```

The other workspaces have a `hash-` prefix after slash
```suggestion
  "name": "@hashintel/hash-task-executor",
```

---

Will change for consistency :) I was a little unsure of if those were things specific to the HASH app (and this doesn't necessarily feel opinionated about it at the moment)

---

Yeah I suppose it isn't tied to the HASH app in particular, but I think it will still be better to do this for consistency

---

That's been done